applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -78,6 +100,18 @@ public static ConfigurationAsyncClientBuilder builder() {
      * Adds a configuration value in the service if that key and label does not exist. The label value of the
      * ConfigurationSetting is optional.
      *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Add a setting with the key "prodDBConnection", label "westUS", and value "db_connection".</p>
+     *
+     * <pre>
+     * client
+     *     .addSetting(new ConfigurationSetting().key("prodDBConnection").label("westUS").value("db_connection"))
+     *     .subscribe(response -&gt; {
+     *         ConfigurationSetting result = response.value();
+     *         System.out.printf("Key: %s, Value: %s", result.key(), result.value());
+     *     });</pre>
```

Comment:
Different code styles between here and line 81 - here you put 'addSetting' on a new line, whereas on line 81 you don't.

Reply:
In my opinion, always have the first operation on the same line, so the code blocks in your JavaDoc that look like this should be improved.

Reply:
All code snippets changed